simply/actor.py

```diff
@@ -228,6 +610,10 @@ def receive_market_results(self, time, sign, energy, price):
         post = (sign * energy, price)
         pre = self.traded.get(time, ([], []))
         self.traded[time] = tuple(e + [post[i]] for i, e in enumerate(pre))
+        # Buying energy as well as pv have positive sign
+        self.matched_energy_current_step += energy*sign
+        # Buying energy therefore decreases the bank
```

Comment:
- in this case, it might be clearer to change the market clearing to use order type signs consistently (we discussed that potentially constants BUY/SELL = +/-1 could make things easier)
- other option would be to change the order type definition in the first place, which might simplify operation on actor side, as it matches the direction of the planned schedule and battery charging rather than the total assets' load